styles/threatconnect.less

```diff
@@ -722,3 +731,160 @@ a.tc-link-tag:hover {
     }
   }
 }
+
+.note-container {
+  max-height: 500px;
+  overflow-y: auto;
+  overflow-x: auto;
+  margin: 6px;
+}
+.note-summary:last-child {
+  border-bottom: none;
+}
+
+.author-information {
+  font-size: 11px;
+  font-weight: 400;
+}
+
+.note-summary {
+  margin: 5px 5px 5px 5px;
+  border-bottom: 1px solid #eee;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  transition: all 0.3s ease;
+}
+
+.note-summary.expanded {
+  display: block;
+  -webkit-line-clamp: unset;
+  overflow: visible;
+  white-space: pre-line;
+}
+
+.view-toggle-wrapper {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: 25px;
+}
+
+.add-integration-data-label {
+  cursor: pointer;
+  padding: 3px;
+  margin: 0px;
+  border-radius: 4px;
+
+  .label-text {
+    padding-left: 5px;
+  }
+
+  &:hover {
+    background-color: #eee;
+  }
+}
+
+.write-container {
+  .integration-labels-description {
+    margin: 5px 0px 5px 3px;
+  }
+
+  .integration-labels-container {
+    max-height: 200px;
+    overflow-y: scroll;
+    border: 1px solid #ccc;
+    padding: 5px;
+    border-radius: 4px;
+    resize: vertical;
+
+    &.disabled {
+      background-color: #f6f6f2;
+    }
+
+    .integration-write-label {
+      cursor: pointer;
+      padding: 3px;
+      margin: 0px;
+      border-radius: 4px;
+
+      .label-text {
+        padding-bottom: 2px;

```

Comment:
This bottom padding is causing the text to be misaligned with the checkbox for me on Windows.  Does it looks misaligned for you too?

<img width="424" height="57" alt="image" src="https://github.com/user-attachments/assets/02d4c1bc-b6a1-4183-8521-f5756cce8d64" />

Remove the `padding-bottom` fixes it since we're already aligning the contents center in the parent flex div.  Would need to check if this affects checkboxes elsewhere though.